src/content/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: 'Node Exporter integration'
+tags:
+  - Integrations
+  - Node Exporter 
+  - Prometheus
+  - host monitoring 
+freshnessValidatedDate: 2024-04-10 
+---
+
+Do you want to monitor hardware and kernel metrics for a Linux server? You can do this with the New Relic remote write integration and the Prometheus Node Exporter. When you combine these two programs with the Prometheus monitoring system, you can send data to New Relic where you can analyze it. 
+
+## Prerequisites [#prerequisites]
+
+Our instructions here are based on the Prometheus guide [Monitoring Linux host metrics with the node exporter](https://prometheus.io/docs/guides/node-exporter/). We'll expand on those instructions with steps to help you send your data to New Relic.
+
+Here's what you need to get started:
+
+* Decide which Linux host you want to instrument. We'll show examples below for Linux servers in EC2, GCP, and Azure instances.
+* Make sure you've installed the Prometheus monitoring system. If you haven't already, you can find download it from the [Prometheus site](https://prometheus.io/download/). 
+
+## Download and start Node Exporter [#download-node-exporter] 
+
+Complete the following:
+
+1. Download and start Node Exporter with the commands below. Be sure to replace the `wget` URL with the latest from the Prometheus [downloads](https://prometheus.io/download#node_exporter) page:
+
+    ```bash
+    # Note that <VERSION>, <OS>, and <ARCH> are placeholders.
+    wget https://github.com/prometheus/node_exporter/releases/download/v<VERSION>/node_exporter-<VERSION>.<OS>-<ARCH>.tar.gz
+    tar xvfz node_exporter-*.*-amd64.tar.gz
+    cd node_exporter-*.*-amd64
+    ./node_exporter
+    ```
+2. Set Node Exporter to run in the background with the keyboard commands `CONTROL + z` and `bg`. In production environments, you'd want to set this up as a service (for example, with `systemd`).
+
+## Configurations [#configs]
+
+Before you start Prometheus, you'll need to make some changes in your main `prometheus.yml` configuration file. We'll start with this basic `prometheus.yml` example below and add configurations to it in the remaining sections. You can copy these examples and paste them into your configuration file.
```

Comment:
This section is really great. I want to borrow this style and format. Really nice work!